Documentation/CodeSnippets/FlexForms/Examples/PluginHaikuList.rst.txt

```diff
@@ -5,6 +5,9 @@
    :caption: EXT:examples/Configuration/Flexforms/PluginHaikuList.xml
 
    <T3DataStructure>
+       <meta>
+           <langDisable>1</langDisable>
```

Comment:
I remember that langDisable was deprecated with TYPO3 v7. If that is in EXT:examples, it should be removed there.

https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/7.6/Deprecation-70138-FlexFormLanguageHandling.html

Reply:
I wonder how it got in there then. Feel free to make a PR there. Next update will then remove these